app/scripts/views/sign_up.js

```diff
@@ -225,6 +227,17 @@ function (Cocktail, _, p, BaseView, FormView, Template, AuthErrors, mailcheck,
       }
       return this._isMailcheckEnabledValue;
     },
+    _wasMailcheckUseful: function () {
+      var email = this.$el.find('.email');
```

Comment:
ok. One thing we should watch out for, I see mailcheck takes [a reference to an element](https://github.com/mozilla/fxa-content-server/blob/908afb88206d7ff8a5497d0b21a829f2cf1eda9e/app/scripts/views/sign_up.js#L231), and then [uses el.val()](https://github.com/mozilla/fxa-content-server/blob/908afb88206d7ff8a5497d0b21a829f2cf1eda9e/app/scripts/lib/mailcheck.js#L40) to get the value. There can be leading whitespace on an email address, `getElementValue` will trim that for email addresses already. Does leading whitespace have any effect on mailcheck?